File: NC_Flights/Shared/Models/Airline.cs
using System;
using System.Collections.Generic;

namespace NC_Flights.Shared;

public partial class Airline
{
    public int Id { get; set; }

    public string Airline1 { get; set; } = null!;

    public short Flight { get; set; }

    public string AirportFrom { get; set; } = null!;

    public string AirportTo { get; set; } = null!;

    public byte DayOfWeek { get; set; }

    public short Time { get; set; }

    public short Length { get; set; }

    public bool Delay { get; set; }
}

File: Node_API/src/Airline.js
class Airline{
    constructor(Id,Airline,Flight,AirportFrom,AirportTo,DayOfWeek,Time,Length,Delay){
        this.Id = Id;
        this.Airline = Airline;
        this.Flight = Flight;
        this.AirportFrom = AirportFrom;
        this.AirportTo = AirportTo;
        this.DayOfWeek = DayOfWeek;
        this.Time = Time;
        this.Length = Length;
        this.Delay = Delay;
    }
}

module.exports = Airline;